README.md

```diff
@@ -27,14 +27,21 @@ The Lounge is the official and community-managed fork of [Shout](https://github.
 ## Installation and usage
```

Comment:
What about changing the destination of the npm version badge above from https://www.npmjs.org/package/thelounge to https://yarn.pm/thelounge? I find Yarn's package page so much nicer.

(I know the actual destination is https://yarnpkg.com/en/package/thelounge, but the `yarn.pm` one is the one they show on page + I'm wondering if they wouldn't redirect to the language requested by the browser)

Reply:
You can tackle this in your revamped readme PR. I'm fine with yarn link.